src/content/docs/new-relic-solutions/best-practices-guides/full-stack-observability/synthetic-monitoring-best-practices-guide.mdx

```diff
@@ -13,115 +13,130 @@ redirects:
 
-To ensure that web services are in place, working as expected, and error-free, you'll want continuous access to the results of your application's performance. Synthetic monitoring gives this type of assurance by performing automated tests on your web application for each selected location. Your synthetic monitors will note downtime instances (called "violations") and collect the aggregated numbers, results, and detailed statistics for each page resource.
+    <img
+    src={syntheticsMonitorCriticalAlert}
+    title="Synthetic monitor critical alert"
+    alt="A screenshot that shows where your critical alerts are located in the UI."
+    />
+    </Step>
 
-To get quickly discover failing monitors, you can use the [index of synthetic monitors](/docs/new-relic-one-monitors-index) page to see which monitors have open violations, success rate over 24 hours, number of failing locations, the monitor period, and monitor type. Clicking on a monitor, you'll be taken to the [Summary page](/docs/synthetics/new-relic-synthetics/pages/synthetics-overview-page-view-monitors-performance) where you'll be presented with information to assess where that particular monitor is failing, why it is failing (last error message, error response codes, duration by domain), and more, to give you a deeper insight into your application's performance over time.
+    <Step>
+    ## Get an overview of your application's performance [#synthetics-monitors]
 
-### View your monitors in your list of entities [#howto-2]
+    To ensure that web services are in place, working as expected, and error-free, you'll want continuous access to the results of your application's performance. Synthetic monitoring gives this type of assurance by performing automated tests on your web application for each selected location. Your synthetic monitors will note downtime instances (called "violations") and collect the aggregated numbers, results, and detailed statistics for each page resource.
 
-<CollapserGroup>
-  <Collapser
-    id="summary-ui"
-    title="Entity explorer"
-  >
-    To view a list of monitors::
+    To get quickly discover failing monitors, you can use the [index of synthetic monitors](/docs/new-relic-one-monitors-index) page to see which monitors have open violations, success rate over 24 hours, number of failing locations, the monitor period, and monitor type. Clicking on a monitor, you'll be taken to the [Summary page](/docs/synthetics/new-relic-synthetics/pages/synthetics-overview-page-view-monitors-performance) where you'll be presented with information to assess where that particular monitor is failing, why it is failing (last error message, error response codes, duration by domain), and more, to give you a deeper insight into your application's performance over time.
```

Comment:
"To get quickly discover" should probably be reworded

Reply:
"success rates over 24 hours"

Reply:
Clicking on a monitor should probably be "When you click on a monitor"

Reply:
"you'll be presented with" > "you'll find information" or "you'll see information"

Reply:
why it is failing > why it's failing